packages/eas-cli/src/build/queries.ts

```diff
@@ -55,18 +55,50 @@ export async function listAndRenderBuildsOnAppAsync(
   }
 }
 
+function formatBuildChoiceValue(value: string | undefined | null): string {
+  return value ? chalk.bold(value) : chalk.dim('Unknown');
+}
+
+function formatBuildChoiceTitleAndDescription(build: BuildFragment): {
+  title: string;
+  description: string;
+} {
+  const splittedCommitMessage = build.gitCommitMessage?.split('\n');
```

Comment:
https://livexp.com/blog/is-it-split-or-splitted-and-some-other-similar-irregular-verbs/

Reply:
Thanks!